components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/EndpointResource.java

```diff
@@ -80,28 +63,24 @@ public boolean invoke(MessageContext messageContext) {
         org.apache.axis2.context.MessageContext axis2MessageContext =
                 ((Axis2MessageContext) messageContext).getAxis2MessageContext();
 
-        List<NameValuePair> queryParameter = getQueryParameters(axis2MessageContext);
+        List<NameValuePair> queryParameter = Utils.getQueryParameters(axis2MessageContext);
 
-        try {
-            // if query params exists retrieve data about specific endpoint
-            if (null != queryParameter) {
-                for (NameValuePair nvPair : queryParameter) {
-                    if (nvPair.getName().equals("endpointName")) {
-                        populateEndpointData(messageContext, nvPair.getValue());
-                    }
+        // if query params exists retrieve data about specific endpoint
+        if (null != queryParameter) {
+            for (NameValuePair nvPair : queryParameter) {
+                if (nvPair.getName().equals("endpointName")) {
+                    populateEndpointData(messageContext, nvPair.getValue());
                 }
-            } else {
-                populateEndpointList(messageContext);
             }
-
-            axis2MessageContext.removeProperty("NO_ENTITY_BODY");
-        } catch (XMLStreamException e) {
-            log.error("Error occurred while processing response", e);
+        } else {
+            populateEndpointList(messageContext);
         }
+
+        axis2MessageContext.removeProperty("NO_ENTITY_BODY");
```

Comment:
We can define a variable for **NO_ENTITY_BODY** also in a util class and use it everywhere . It is also being used frequently . WDYT